plugin/ochttp/server.go

```diff
@@ -130,7 +130,7 @@ func (h *Handler) startTrace(w http.ResponseWriter, r *http.Request) (*http.Requ
 		}
 	}
 	span.AddAttributes(requestAttrs(r)...)
-	if r.Body == nil {
+	if r.Body == http.NoBody {
```

Comment:
```suggestion
	if r.Body == nil || r.Body == http.NoBody {
```

lets keep the nil-check, just in-case.